python/src/eval.cc

```diff
@@ -0,0 +1,72 @@
+#include "internal/eval.hh"
+#include "internal/errors.hh"
+#include "internal/nix-to-python.hh"
+#include "internal/python-to-nix.hh"
+#include <Python.h>
+
+#include <cxxabi.h>
+#include <store-api.hh>
+#include <shared.hh>
+
+namespace pythonnix {
+
+const char *currentExceptionTypeName() {
+  int status;
```

Comment:
I could configure a pre-commit hook and ci check for a subset of files without much effort.
 - #7634

Reply:
It looks like @roberth since opened https://github.com/NixOS/nix/pull/7745 to only check a subset of files, but @edolstra [thinks](https://github.com/NixOS/nix/pull/7745#issuecomment-1422451469) that #6721 would be better.

I'd rather avoid making this PR dependent on these decisions, so for now I just formatted all the files using the `.clang-format` config file from the above-linked PR's (it's the same for both).

Reply:
Much appreciated, @infinisil.

This conversation would not have been needed if #7745 was merged.